package.json

```diff
@@ -38,27 +38,27 @@
   },
   "devDependencies": {
     "7zip-bin": "^2.0.4",
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.3.2",
+    "babel-core": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.0.0",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-react": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
     "chai": "^3.5.0",
-    "cross-env": "^3.1.4",
+    "cross-env": "^4.0.0",
     "devtron": "^1.4.0",
-    "electron": "1.6.2",
-    "electron-builder": "^14.5.3",
-    "electron-builder-squirrel-windows": "^15.0.0",
+    "electron": "1.6.6",
```

Comment:
Do we want to wait for a bit until we update to 1.6.8? That version of Electron was needed for this PR https://github.com/mattermost/desktop/issues/341